sympy/galgebra/printing.py

```diff
@@ -30,10 +30,14 @@
 
 import os
 import sys
+
```

Comment:
I doubt there is a PEP8 guideline for that.
The justification does not refer to PEP8 anyway.